clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -300,18 +307,18 @@ public void testJoinGroupInvalidGroupId() {
     public void testNormalJoinGroupLeader() {
         final String consumerId = "leader";
 
-        subscriptions.subscribe(singleton(topicName), rebalanceListener);
+        subscriptions.subscribe(Pattern.compile("test.*"), rebalanceListener);
 
         // ensure metadata is up-to-date for leader
-        metadata.setTopics(singletonList(topicName));
+        metadata.setTopics(singletonList(topic1));
         metadata.update(cluster, time.milliseconds());
 
         client.prepareResponse(groupCoordinatorResponse(node, Errors.NONE.code()));
         coordinator.ensureCoordinatorReady();
 
         // normal join group
-        Map<String, List<String>> memberSubscriptions = Collections.singletonMap(consumerId, singletonList(topicName));
-        partitionAssignor.prepare(Collections.singletonMap(consumerId, singletonList(tp)));
+        Map<String, List<String>> memberSubscriptions = Collections.singletonMap(consumerId, singletonList(topic1));
+        partitionAssignor.prepare(Collections.singletonMap(consumerId, Arrays.asList(t1p, t2p)));
```

Comment:
Related to comment above. Can we do this in a separate test case? This is not the "normal" case, but it should be covered somewhere.